testdata/p4_16_samples_outputs/parser-inline/parser-inline-test1-frontend.p4

```diff
@@ -70,7 +70,29 @@ parser ParserImpl(packet_in packet, out headers hdr, inout metadata meta, inout
         hdr.h2.setInvalid();
         hdr.h3.setInvalid();
         hdr.h4.setInvalid();
-        transition Subparser_start;
+        transition Subparser_start_0;
```

Comment:
I do not understand why this happens, but this seems like correct behavior to me? The parser is actually inlined. But this is something we should properly check.